x-pack/plugins/ml/public/application/overview/components/anomaly_detection_panel/table.tsx

```diff
@@ -68,8 +58,14 @@ export const AnomalyDetectionTable: FC<Props> = ({ items, jobsList, statsBarData
   const [sortField, setSortField] = useState<string>(AnomalyDetectionListColumns.id);
   const [sortDirection, setSortDirection] = useState<Direction>('asc');
 
+  const timeBuckets = useTimeBuckets();
+
+  const manageJobsLink = useMlLink({
+    page: ML_PAGES.ANOMALY_DETECTION_JOBS_MANAGE,
+  });
+
   // columns: group, max anomaly, jobs in group, latest timestamp, docs processed, action to explorer
-  const columns: AnomalyDetectionTableColumns = [
+  const columns: Array<EuiBasicTableColumn<Group>> = [
```

Comment:
The latest timestamp column needs more width as the time is always being truncated. Could probably take width off the `Docs processed` column.

![image](https://user-images.githubusercontent.com/7405507/146040835-cd0ddf3e-86cc-4e10-bf67-190ab9844679.png)

Reply:
Fixed in 49b1f13a92514bf49c96cee349278eae040b94bf

Reply:
The width looks good, but now I'm wondering if we need to show the timestamp down to the second level? Maybe minute level is sufficient detail for the Overview table?

Reply:
Done in 84caa57dc55070fc2e778512b214f130d04506ca